test/http-test.js

```diff
@@ -139,7 +140,7 @@ describe('HTTP', function() {
 
     assert(receive);
     assert.strictEqual(receive.name, 'default');
-    assert.strictEqual(receive.type, 'pubkeyhash');
+    assert.strictEqual(receive.type, 'witness');
```

Comment:
I think we should have both test cases, should not be hard to have both legacy and witness wallet tests.